client/scripts/helpers/index.ts

```diff
@@ -343,3 +343,32 @@ export const removeOrAddClasslistToAllElements = (
     return METHODS[method]();
   });
 };
+
+export const expandTokenAmount = (input: string, decimals: number) : string => {
+  // necessary unfortunately because BN.js can't parse decimal strings
+  const parts = input.split('.');
+  const zeroesToAdd = parts[1] ? decimals - parts[1].length : decimals;
+
+  if (zeroesToAdd < 0) { throw new Error('More decimals supplied than are'); }
+  return parts[0] + (parts[1] ? parts[1] : '') + '0'.repeat(zeroesToAdd);
+};
```

Comment:
What is the data source for this? Why are these methods `expandTokenAmount` and `formatTokenAmount` necessary?

@jnaviask

Reply:
It should always be an integer, correct? I don't see what's up with the decimals.

Reply:
I'll make these function names less awkward. We are converting from value denominated in tokens to value denominated in token base units and vice versa. When a user (admin) is typing on the front end how many tokens a threshold should be, it could be something like `0.01`. This gets converted to token base units before being stored in the back end